web/generate-geometries.js

```diff
@@ -531,13 +533,14 @@ const zoneDefinitions = [
   { zoneName: 'NO-NO4', type: 'subZone', id: 'NO-NO4' },
   { zoneName: 'NO-NO5', type: 'subZone', id: 'NO-NO5' },
   { zoneName: 'NP', type: 'country', id: 'NPL'},
-  // { zoneName: 'NZ', type: 'country', id: 'NZL'},
   { zoneName: 'NU', type: 'country', id: 'NIU'},
   { zoneName: 'NR', type: 'country', id: 'NRU'},
+  // { zoneName: 'NZ', type: 'country', id: 'NZL'},
```

Comment:
Any reason for keeping this? Can it be useful in the future?

Reply:
As discussed, this is what we've generally done for all other zones to indicate that we're using subzones instead of the country :)